internal/core/metadata/application/notify.go

```diff
@@ -138,6 +118,11 @@ func publishSystemEvent(eventType, action, owner string, dto any, ctx context.Co
 			detailName = pw.Name
 		} else {
 			lc.Errorf("can not convert to provision watcher DTO")
+	case common.DeviceServiceSystemEventType:
+		if service, ok := dto.(dtos.DeviceService); ok {
+			detailName = service.Name
```

Comment:
DetailName is set but never used.

Reply:
It's used for the logs.

https://github.com/edgexfoundry/edgex-go/blob/c89f0abc00fef79390aa7d6b0c8e6f8da691ba97/internal/core/metadata/application/notify.go#L158
https://github.com/edgexfoundry/edgex-go/blob/c89f0abc00fef79390aa7d6b0c8e6f8da691ba97/internal/core/metadata/application/notify.go#L162

Reply:
Got it. THX!